ts/store/reducers/entities/services/servicesById.ts

```diff
@@ -25,9 +25,12 @@ const reducer = (
   switch (action.type) {
     case getType(loadService.request):
       // Use the ID as object key
+      const prevServiceRequest = state[action.payload] || pot.noneLoading;
```

Comment:
we are requesting for a refresh, why should the pot be in error?
```typescript
const cachedValue = state[action.payload];
      const prevServiceRequest =
        cachedValue && pot.isSome(cachedValue) && !pot.isLoading(cachedValue)
          ? pot.someLoading(cachedValue.value)
          : pot.noneLoading;

return {
        ...state,
        [action.payload]: prevServiceRequest
      };
```

Reply:
The use of someLoading causes this behavior for this reason I had managed the refresh request differently.

![someLoading](https://user-images.githubusercontent.com/2670647/66570374-c1fc2280-eb6d-11e9-8f00-7ac1b7045f8d.gif)

Reply:
Yes I know this behavior. This will be changed in a future PR. When we ask for a service loading the pot should be in loading status